code/game/objects/items/cigs_lighters.dm

```diff
@@ -164,14 +168,17 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		if(lighting_text)
 			if(src.reagents.get_reagent_amount(/datum/reagent/toxin/plasma))
 				message_admins("[src] that contains plasma was lit by [ADMIN_LOOKUPFLW(user)]!")
-				log_game("[src] that contains plasma was lit by  [key_name(user)]!")
+				log_game("[src] that contains plasma was lit by [key_name(user)]!")
```

Comment:
if you could add another line for user.log_message it makes this easier to spot ingame logs

Reply:
cigar code is written shit and that'll be better at refactoring it next time.